packages/scripts/src/remove-cases.js

```diff
@@ -275,6 +277,9 @@ const removeCase = async (reference) => {
 		const startTime = Date.now();
 		await databaseConnector.$transaction(
 			async (tx) => {
+				if (!reference.startsWith('TRAIN')) {
```

Comment:
I am wondering, if we should be filtering using the sector "Training", rather than filtering based on a string compare on the case reference?  As that is the sector for all Training cases.

Reply:
Discussed and resolved